health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -545,9 +576,35 @@ const ViewHierarchy = () => {
+                                {!disableFile && 
+                                  <div style={{ display: "flex", justifyContent: "flex-end", alignItems: "center", width: "100%" }}>
+                                    <div style={{ display: "flex", justifyContent: "space-between", alignItems: "center", maxWidth: "20rem", gap:"1rem" }}>
+                                      <div style={{ marginRight: "10px" }}>
+                                        {fileName}
+                                      </div>
+                                      <div
+                                        className="dustbin-icon"
+                                        onClick={() => removeFile()}
+                                        onKeyDown={(e) => { if (e.key === 'Enter') removeFile(); }}
+                                        tabIndex="0"
+                                        role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)